### ISSUE TITLE
Make eye-catchers explicit

### ISSUE THREAD

**Issue by bartlomieju** [denoland/deno#13001]
Through Deno codebase we got a bunch of so called "eye-catchers", ie. prompt messages that get printed during lifecycle of the problem. Most popular eye-catchers are `Download ...` and `Check ...` messages that users see when first downloading dependencies and type checking the code. 

Unfortunately these messages are scattered all across the codebase as simple `log::info!()` calls. We should centralise them in a single module (`cli/eye_catchers.rs`) to make them more explicit and easier to change should the need arise.

Here are some examples in the code that print these messages:

https://github.com/denoland/deno/blob/d31378726e78490e88b8a9ec3001b86ea009d978/cli/main.rs#L725

https://github.com/denoland/deno/blob/d31378726e78490e88b8a9ec3001b86ea009d978/cli/proc_state.rs#L518-L522

https://github.com/denoland/deno/blob/d31378726e78490e88b8a9ec3001b86ea009d978/cli/file_watcher.rs#L110-L113

https://github.com/denoland/deno/blob/d31378726e78490e88b8a9ec3001b86ea009d978/cli/file_watcher.rs#L159-L162

**Comment by jopefd** [comment]
I'm thinking of creating functions like `eye_catchers::check` or `eye_catchers::emit`, but I need to know all eye-catchers to create all these functions. Since I realize that the eye-catchers are in green, I could do this using something like `git grep "colours::green(`?

**Comment by jopefd** [comment]
I'm thinking of creating functions like `eye_catchers::check` or `eye_catchers::emit`, but I need to know all eye-catchers to create all these functions. I could do this using something like `git grep "info!("`?

**Comment by bartlomieju** [comment]
> I'm thinking of creating functions like eye_catchers::check or eye_catchers::emit, but I need to know all eye-catchers to create all these functions. I could do this using something like git grep "info!("?

Yes, this should be enough for first pass. I suggest to skip `cli/lsp/` directory as info logs there are debug logs and should be left as they are.

> In addition, would I pass a list of arguments to these functions? I ask this because some calls have two arguments, and others have more, like:

Yes, but instead of passing a list I suggest to pass arguments explicitly.
